tee-worker/omni-executor/parentchain/primitives/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "parentchain-primitives"
+version = "0.1.0"
+authors = ['Trust Computing GmbH <info@litentry.com>']
+edition.workspace = true
+
+[dependencies]
+executor-core = { path = "../../executor-core" }
```

Comment:
Do you consider using workspace dep?

Reply:
We probably should. I was just trying to be consistent with what we currently have. I'll update the dependencies 👍🏼